src/tbots.py

```diff
@@ -8,6 +8,12 @@
 from thefuzz import fuzz
 from thefuzz import process
 
+# thefuzz is a fuzzy string matcher in python
+# https://github.com/seatgeek/thefuzz
+#
+# It returns a match ratio between the input and the choices
+# This is an experiemntally determined threshold that works
```

Comment:
Last comment, nit: experimentally misspelled.

Reply:
lol I can't spell for shit, fixed

Reply:
I feel that aswell, you need a spell checker😅